### PR TITLE
Controller revert message

### PR DESCRIPTION
**Pull request by i-stam**
- test revert statements

### CODE REVIEW

**Comment by i-stam** [comment]
why `onlyAdmin() `can claim and not `onlyAdminOrOwner()`?

File: contracts/controller.sol [hunk 1]
```diff
@@ -61,13 +61,13 @@ contract Controller is IController, Ownable, Transferrable {
 
     /// @notice Checks if message sender is an admin.
     modifier onlyAdmin() {
-        require(_isAdmin[msg.sender], "sender is not an admin");
```
Comment (nostdm) [inline contracts/controller.sol:64]: Not sure if it's better with bad grammar :-)

File: test/controller/add_admin_test.go [hunk 2]
```diff
@@ -47,151 +47,71 @@ var _ = Describe("addAdmin", func() {
...
-		})
-
-		It("should not exaust gas", func() {
-			Expect(isGasExhausted(tx, gasLimit)).To(BeFalse())
-		})
-
-		It("should fail at already controller requirenment", func() {
-			Expect(TestRig.LastExecuted()).To(MatchRegexp(`require\(!_isController\[_account\], "provided account is already a controller"\);`))
+			returnData, _ := ethCall(tx)
+			Expect(string(returnData[len(returnData)-64:])).To(ContainSubstring("provided account is already a controller"))
```
Comment (nostdm) [inline test/controller/add_admin_test.go:68]: These expected strings have the a/an articles - why remove them from only two require statements?

**Comment by i-stam** [comment]
In general, the revert strings were chopped because they reduce the binary significantly and it was a way to avoid the wallet-deployer deployment error we were getting. I noticed that in the other statements there are articles but I think we can edit all those when V4 is merged and we have space to make them grammatically correct too. I don't have strong opinions on that, I left it up to you guys. :)

**Pull request merged**